screens/NewPostImage.tsx

```diff
@@ -0,0 +1,270 @@
+import React, { useEffect, useRef, useState } from "react";
+import Modal from "react-native-modal";
+
+import {
+  StyleSheet,
+  Text,
+  Pressable,
+  View,
+  Image,
+  Alert,
+  TouchableOpacity,
+  ImageBackground,
+  Dimensions,
+  Platform,
+  ListRenderItem,
+  StatusBar,
+} from "react-native";
+const { width: screenWidth } = Dimensions.get("window");
+import * as ImagePicker from "expo-image-picker";
+
+import { AntDesign, Feather } from "@expo/vector-icons";
+import { pressedOpacity } from "../constants/Values";
+import Carousel, {
+  Pagination,
+  ParallaxImage,
+} from "react-native-snap-carousel";
+import AnimatedDotsCarousel from "react-native-animated-dots-carousel";
+
+export function NewPostImage({ navigation }) {
+  const [image, setImage] = useState([]);
+  const [activeSlide, setActiveSlide] = useState(0);
+  const [refresh, setFresh] = useState(false);
+  const _carousel = useRef(null);
+
+  const pickImage = async () => {
+    let result = await ImagePicker.launchImageLibraryAsync({
+      mediaTypes: ImagePicker.MediaTypeOptions.All,
+      allowsEditing: true,
+      aspect: [4, 3],
+      quality: 1,
+    });
+
+    if (!result.cancelled) {
+      setImage([...image.slice(0, -1), result.uri, ""]);
+    }
+  };
+  function _renderItem({ item, index }, parallaxProps) {
+    return item == "" ? (
+      <View
+        style={[
+          styles.item,
+          { justifyContent: "center", alignItems: "center" },
+        ]}
+      >
+        <TouchableOpacity
+          activeOpacity={pressedOpacity}
+          style={styles.roundButton1}
+          onPress={() => {
+            pickImage();
+          }}
+        >
+          <Feather name="plus" size={36} color="black" />
+        </TouchableOpacity>
+      </View>
+    ) : (
+      <View style={styles.item}>
+        <ParallaxImage
+          source={{ uri: item }}
+          containerStyle={styles.imageContainer}
+          style={styles.image}
+          parallaxFactor={0.4}
+          showSpinner={true}
+          spinnerColor={"rgba(255, 255, 255, 0.4)"}
+          {...parallaxProps}
+        />
+        <TouchableOpacity
+          activeOpacity={pressedOpacity}
+          style={{ position: "absolute", bottom: 20, left: 20 }}
+          onPress={() => {
+            if (image.length == 2) {
+              setImage([]);
+            } else {
+              image.splice(index, 1);
+              setImage(image);
+              setFresh(!refresh);
+            }
+          }}
+        >
+          <Feather name="trash" size={28} color="white" />
```

Comment:
trash doesn't work rn right? if not we should comment this out otherwise people will think it's a bug